addons/web/static/src/views/basic_relational_model.js

```diff
@@ -293,6 +293,11 @@ export class Record extends DataPoint {
         return true;
     }
 
+    isInvisible(fieldName) {
+        const { invisible } = activeField.modifiers || {};
+        return invisible ? evalDomain(invisible, this.evalContext) : false;
+    }
+
```

Comment:
This is only required in a form view isn't it?
The problem was coming from the list view no?
```suggestion
```